### ISSUE TITLE
Does not fix W605 in multilines

### ISSUE THREAD

**Issue by belugame** [hhatto/autopep8#448]
Latest flake8 3.6 complains about many `W605 invalid escape sequence '\d'` in my code.
I use python 2.7.6 and tried to fix the problems with autopep8, but it did not find or fix them.

Reproducible with:

```
regex = '\d+(\.\d+){3}$'
foo = validators.RegexValidator(
     regex='\d+(\.\d+){3}$')
```
`autopep8 --ignore-local-config  --in-place --aggressive regex.py`

It will fix the first one but not the 2nd one.

**Comment by belugame** [comment]
Relevant information may be found here https://github.com/PyCQA/pycodestyle/issues/755

**Comment by asottile** [comment]
If you're looking for another tool in the meantime (which doesn't depend on pycodestyle) to fix these, [pyupgrade](https://github.com/asottile/pyupgrade#invalid-escape-sequences) has a fixer for this

**Comment by hhatto** [comment]
We have released version 1.5.5.
This version should fix the problem.
